api-client/src/endpoints/createPOIsEndpoint.ts

```diff
@@ -38,7 +38,9 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<PoiModel>> =>
               region: poi.location.region,
               country: poi.location.country,
               latitude: poi.location.latitude,
-              longitude: poi.location.longitude
+              longitude: poi.location.longitude,
+              thumbnail: poi.thumbnail,
+              path: normalizePath(poi.path)
```

Comment:
Same as above.

Reply:
not need when we pass it view convertToPoint